planty/users/templates/users/logout.html

```diff
@@ -0,0 +1,18 @@
+{% extends "planty/base.html" %}
+
+{% block content %}
+
```

Comment:
NIT: we can delete unnecessary empty lines after `{% block content %}` and before `{% endblock content %}` (and maybe also clean up other templates the same way).